web/src/message_events.js

```diff
@@ -237,6 +239,7 @@ export function update_messages(events) {
                     anchor_message.topic,
                 );
                 recent_topics_ui.inplace_rerender(topic_key);
+                inbox_ui.update();
```

Comment:
What's the thinking on not passing something like a `topic_key` to this call? 

Even if the `inbox_ui` module ignores the parameter, it might set better expectations about the intent here. I'm not sure ... it's possible the inbox view is something that'll be a relatively near term target for trying to rebuild using Solid, since it depends on less data than a lot of others.